### PR TITLE
Fix bug with alignment for root level decorator nodes v2

### PR DESCRIPTION
**Pull request by thegreatercurve**
Should fix https://github.com/facebook/lexical/issues/1696.

https://user-images.githubusercontent.com/7748470/163989526-98345536-3dbb-43b9-9bd7-63ecb5923b38.mov

### CODE REVIEW

File: packages/lexical-playground/src/nodes/TweetNode.jsx [hunk 1]
```diff
@@ -105,7 +111,18 @@ export class TweetNode extends DecoratorNode<React$Node> {
   }
 
   decorate(): React$Node {
-    return <TweetComponent loadingComponent="Loading..." tweetID={this.__id} />;
+    return (
+      <TweetComponent
```
Comment (zurfyx) [inline packages/lexical-playground/src/nodes/TweetNode.jsx:115]: Should we create a BlockComponent wrapper on top that handles the container bits for you automatically? This layer can also listen to `FORMAT_TEXT` commands which would solve centering when clicking the node and can be reused within Twitter and Youtube and possibly others soon (so I'd even consider making it part of @lexical/react)

**Comment by thegreatercurve** [comment]
Found another related issue whilst fixing this one: #1868

File: packages/lexical-playground/src/nodes/TweetNode.jsx [hunk 2]
```diff
@@ -72,15 +74,19 @@ function TweetComponent({
...
+        style={{display: 'inline-block', width: '550px'}}
+        ref={containerRef}
+      />
+    </div>
   );
 }
 
 export class TweetNode extends DecoratorNode<React$Node> {
   __id: string;
+  __format: ?ElementFormatType;
```
[the remark this answers is not in the document]
Comment (trueadm) [reply packages/lexical-playground/src/nodes/TweetNode.jsx:89]: Why put it on DecoratorNode? Inline decorators wouldn't use it? It's best to keep it here IMO.

File: packages/lexical-playground/src/nodes/TweetNode.jsx (hunk 1 above)
Comment (trueadm) [reply packages/lexical-playground/src/nodes/TweetNode.jsx:115]: That's a good shout!

Comment (thegreatercurve) [reply packages/lexical-playground/src/nodes/TweetNode.jsx:115]: I think this and the above both seem like sensible suggestions. What do you think @trueadm? This probably has implications for the work you mentioned last week about adding fake cursors for `DecorateNodes`, as they're not currently focusable by themselves?

Comment (trueadm) [reply packages/lexical-playground/src/nodes/TweetNode.jsx:115]: You should be adding selection to embeds, in a similar way to how ImageNode and ExcalidrawNode has the ability to work with NodeSelection.

**Review by trueadm — APPROVED** [review]

File: packages/lexical-playground/src/nodes/TweetNode.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (thegreatercurve) [reply packages/lexical-playground/src/nodes/TweetNode.jsx:89]: If we add any more instances might be worth adding an intermittent extension of `AlignableDecoratorNode`/`BlockDecoratorNode`, but probably overkill in the mean time.

**Pull request merged**